sphinx/writers/latex.py

```diff
@@ -1378,19 +1360,25 @@ def depart_hlistcol(self, node):
         pass
 
     def latex_image_length(self, width_str):
-        match = re.match('(\d*\.?\d*)\s*(\S*)', width_str)
+        return self.latex_dim_to_latexdim(width_str)
+
+    def latex_dim_to_latexdim(self, length_str):
```

Comment:
Note:
I noticed that this method does not use `self` argument. It is not needed to be a method.
To change this to function is much better.